aat/src/aat/resources/features/F-097 RoleToAccessProfiles/S-097.8.td.json

```diff
@@ -0,0 +1,57 @@
+{
+  "_guid_": "S-097.8",
+  "_extends_": "Get_CaseType",
+
+  "title": "Return the new CaseAccessCategories column as part of the Get RoleToAccessProfiles operation from Definitions store",
+
+  "specs": [
+    "an active profile in CCD",
+    "Get RoleToAccessProfiles",
+    "Definition store API",
+    "contains the newly defined field - CaseAccessCategories"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "id": "FT_CaseAccessCategories"
+    }
+  },
+
+  "expectedResponse": {
+    "body": {
+      "id": "FT_CaseAccessCategories",
+      "description": "Case Type for testing CaseAccessCategories",
+      "version": {
+        "number": "[[ANY_INTEGER_NOT_NULLABLE]]",
+        "live_from": "[[ANYTHING_PRESENT]]",
+        "live_until": null
+      },
+      "name": "[[ANYTHING_PRESENT]]",
+      "jurisdiction": "[[ANYTHING_PRESENT]]",
+      "events": "[[ANYTHING_PRESENT]]",
+      "states": "[[ANYTHING_PRESENT]]",
+      "acls": "[[ANYTHING_PRESENT]]",
+      "searchAliasFields": [],
+      "roleToAccessProfiles": [
```

Comment:
This test looks like a good opportunity to verify my comment on the ticket too - could we try adding more rows to the configuration please e.g. another for the same case type where `role_name` is also `Role1` and there's different `case_access_categories`/`access_profiles` values?

Reply:
This is now going to be done in [RDM-13109](https://tools.hmcts.net/jira/browse/RDM-13109)